frontend/packages/dev-console/src/plugin.tsx

```diff
@@ -353,6 +355,33 @@ const plugin: Plugin<ConsumedExtensions> = [
       getKebabActionsForKind,
     },
   },
+  {
+    type: 'YAMLTemplate',
+    properties: {
+      model: PipelineModel,
+      template: `
```

Comment:
Should the template be moved out to a separate file

Reply:
Probably not a bad idea... I'll create a templates file or something in `dev-console` and export this out of it.

Reply:
Pushed to a new folder structure called `templates` where I created `pipeline.ts` to hold pipeline related templates.